### PR TITLE
Fix the color and alignment of the `checksec` output

### LINKED ISSUE #2196: `checksec` output has incorrect colors and is misaligned.
While playing CTF last weekend, I noticed something odd with the `checksec` output.

Here's the command I used to compiled the binary: `gcc -fno-stack-protector -zexecstack -no-pie -z lazy a.c`.

This is the output I got with `checksec`:

SCR-20240529-uesc

I expected `Partial RELRO` and `NX unknown` to be yellow, and both `Executable` and `Has RWX segments` to be red.
Additionally, the alignment of `File` and `Arch` could be improved.

I was expecting something like this:
SCR-20240529-ufej

### PR DESCRIPTION
**Pull request by lebr0nli**
This PR should resolve #2196 

Before the fix:
SCR-20240529-uesc

After the fix:
SCR-20240529-ufej

### CODE REVIEW

**Comment by disconnect3d** [comment]
Lgtm.

@lebr0nli Should we remove 'draft' from this? 

cc: @peace-maker @Arusekk perhaps this should be fixed/changed on pwntools side of things (?)

**Comment by lebr0nli** [comment]
> Should we remove 'draft' from this?

Sure, I think it's ready now.

File: pwndbg/wrappers/checksec.py [hunk 1]
```diff
@@ -1,13 +1,38 @@
...
 
 def get_raw_out(local_path: str) -> str:
     elf = ELF(local_path)
-    output = f"File: {elf.path}\n"
-    output += f"Arch: {elf.arch}\n"
-    output += elf.checksec()
+    # 10 is the magic number used in elf.checksec() to align the output.
+    # https://github.com/Gallopsled/pwntools/blob/f046fdd93e154bd892332f38cfbb518de130f1f2/pwnlib/elf/elf.py#L2012
+    # We might need to update this number if the implementation of elf.checksec() changes in the future.
+    output = "File:".ljust(10) + elf.path + "\n"
```
Comment (lebr0nli) [inline pwndbg/wrappers/checksec.py:32]: btw I just noticed that the space alignment changes to 12 in pwntools 4.13.0beta0:
https://github.com/Gallopsled/pwntools/blob/a06d378e8cbb054d87220ed310fe145aa622561a/pwnlib/elf/elf.py#L2065
We will need to update this when we bump the pwntools version.

**Comment by peace-maker** [comment]
> cc: @peace-maker @Arusekk perhaps this should be fixed/changed on pwntools side of things (?)

It appears to be a problem with the wrapper command in pwndbg. Running checksec from the command line shows different colors.

grafik

I guess there is nothing to be done in pwntools?

**Pull request merged**